resources/FeatureCatalogItemResource.json

```diff
@@ -23,7 +24,7 @@
                         "type": "number"
                     },
                     "description": "Array that is the bounding box coordinates as per GeoJSON",
-                    "nullable": true
+                    "type": ["string", "null"]
```

Comment:
I don't think this is right. Above "items" is the "type" specifier, which would need to be ["array", "null"] providing this lints ok.